scripts/commit-message

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -e
+
+fail() {
+  (echo && sed 's/^/! /' <<<"$1" && echo) >&2
+  exit 1
+}
+
+# Find the number of the most recently opened pull request merged so far.
+# Add 1 to this to get the smallest possible number of the next pull request,
+# then add an arbitrary buffer to allow for issues and pull requests opened
+# since the creation of the most recently opened pull request merged so far.
```

Comment:
I should be more specific; You describe an algorithm, but I don't understand its purpose.